ui/js/component/video/internal/play-button.jsx

```diff
@@ -4,12 +4,23 @@ import Link from "component/link";
 import Modal from "component/modal";
 
 class VideoPlayButton extends React.PureComponent {
+  componentDidMount() {
+    document.addEventListener("keydown", this.onKeyDown.bind(this));
+  }
+
   onPurchaseConfirmed() {
     this.props.closeModal();
     this.props.startPlaying();
     this.props.loadVideo(this.props.uri);
   }
 
+  onKeyDown(event) {
+    if (event.keyCode === 32) {
```

Comment:
Can we use a code or constant here? Possible https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code?